README.md

```diff
@@ -14,13 +14,23 @@ The library can be loaded from SQLite as such:
 
 ### Docker
 
-A Docker image has been provided to simplify integrating the library into larger projects. When run, an interactive SQLite session is started with the library pre-loaded for testing.
+A Docker image has been provided to simplify integrating the library into larger projects.
```

Comment:
"A Docker image has been provided containing the library statically linked against its dependencies, to simplify integrating the library into larger projects. An additional image is provided as an example of its usage"